src/schema/schema.graphql

```diff
@@ -2745,6 +2745,7 @@ input PreferencesInput {
 	zimbraPrefPowerPasteEnabled: Boolean
 	zimbraPrefDisplayTimeInMailList: Boolean
 	zimbraPrefPrimaryTwoFactorAuthMethod: String
+	zimbraPrefDeleteInviteOnReply: Boolean
```

Comment:
we need to add 'zimbraPrefDeleteInviteOnReply' here https://github.com/Zimbra/zm-api-js-client/blob/master/src/schema/schema.graphql#L1679

Reply:
'zimbraPrefDeleteInviteOnReply' is already defined in Preferences at line 1638. API required ''zimbraPrefDeleteInviteOnReply' under PreferencesInput as well. So added it here.